themes/medius/templates/includes/header.html

```diff
@@ -1,30 +1,136 @@
-    <ul>
-      {% if DISPLAY_CATEGORIES_ON_MENU %}
-        {% for cat, null in categories %}
-          {% if cat != ALL_ARTICLES_LABEL and cat != 'pt' %}
-            <li class="nav-item {% if cat == category %} selected {% endif %}">
-              <a href="{{ SITEURL }}/{{ cat.url }}">{{ cat }}</a>
-            </li>
-          {% endif %}
-        {% endfor %}
-      {% endif %}
+  <div class="header__menu">
+    <ul class="primary_menu_list">
+      <li>
+        <a class="link link--nav link--nav--purple-darker" href="https://labcodes.com.br/about" title="Go to about us page.">About us</a>
+      </li>
+      <li>
+        <a class="link link--nav link--nav--purple-darker" href="https://labcodes.com.br/services" title="Go to services page.">Services</a>
+      </li>
+      <li>
+        <a class="link link--nav link--nav--purple-darker" href="https://labcodes.com.br/careers" title="Go to careers page.">Careers</a>
+      </li>
+      <li>
+        <a class="link link--nav link--nav--purple-darker link--nav--is-active" href="https://labcodes.com.br/blog/" target="_blank" title="Go to our blog page.">Blog</a>
+      </li>
     </ul>
-  </nav>
 
-  <div class="box">
-    <a class="btn-site" href="https://labcodes.com.br" target="_blank" ref="noopener noreferrer">
-      <p>visit labcodes.com.br</p>
-      <span></span>
-    </a>
+    <ul class="secundary_menu_list hidden">
+      <li>
+        <a class="link link--nav link--nav--purple-dark" href="https://labcodes.com.br/about" title="Go to about us page.">About us</a>
+      </li>
+      <li>
+        <a class="link link--nav link--nav--purple-dark" href="https://labcodes.com.br/services" title="Go to services page.">Services</a>
+      </li>
+      <li>
+        <a class="link link--nav link--nav--purple-dark" href="https://labcodes.com.br/careers" title="Go to careers page.">Careers</a>
+      </li>
+      <li>
+        <a class="link link--nav link--nav--purple-dark link--nav--is-active" href="https://labcodes.com.br/blog/" target="_blank" title="Go to our blog page.">Blog</a>
+      </li>
+    </ul>
+  </div>
 
-    <button class="btn-menu" id="btn-open-close"></button>
+  <div class="header__cta">
+    <a class="button small yellow-30" href="https://labcodes.com.br/contact" title="Go to contact page.">get an estimate</a>
+  </div>
+</header>
+
+
+
+<nav id="menu" class="menu-expanded">
+  <div class="menu-expanded__header">
+    <div class="menu-expanded__logo">
+      <img src="/theme/images/icons/header/logo-marca-yellow.svg" alt="Labcodes logo mark">
+    </div>
+    <div class="menu-expanded__close">
+      <a id="menu-expanded__close" type="button" title="Close header menu">
+        <img class="btn-close" src="/theme/images/icons/header/close-button.svg" alt="Close Icon">
+      </a>
+    </div>
+  </div>
+
+  <div class="menu-expanded__links">
+    <ul>
+      <li>
+        <a class="link link--nav" href="https://labcodes.com.br/about" title="Go to about us page.">About us</a>
```

Comment:
Also it's missing HOME link

```suggestion
        <a class="link link--nav" href="https://labcodes.com.br" title="Go to home page.">Home</a>
      </li>
      <li>
        <a class="link link--nav" href="https://labcodes.com.br/about" title="Go to about us page.">About us</a>
```